catalog/spatial/geocoding/spatial-geocoding-lucene/src/main/java/org/codice/ddf/spatial/geocoding/index/GeoNamesLuceneIndexer.java

```diff
@@ -229,10 +228,10 @@ private void addDocument(final IndexWriter indexWriter, final GeoEntry geoEntry,
                 geoEntry.getName(),
                 Field.Store.YES));
 
-        document.add(new DoubleField(GeoNamesLuceneConstants.LATITUDE_FIELD,
+        document.add(new LegacyDoubleField(GeoNamesLuceneConstants.LATITUDE_FIELD,
```

Comment:
It does not look like we query these fields and they are only used to get the original Lat/Lon value.  I think you can just change these to `StoreField` instead of `LegacyDoubleField`.

https://lucene.apache.org/core/6_0_0/core/org/apache/lucene/document/StoredField.html